src/Email.php

```diff
@@ -2,22 +2,55 @@
 
 namespace Drupal\share_light;
 
+/**
+ * A channel for sharing via e-mail.
+ */
 class Email extends ChannelBase {
-  public static function title() { return t('Email'); }
+
+  /**
+   * Returns the channel's name.
+   *
+   * @return string
+   *   Returns t('Email').
+   */
+  public static function title() {
+    return t('Email');
+  }
+
+  /**
+   * Returns the default values for the channel.
+   */
+  public static function defaults() {
+    return [
+      'mailto_toggle' => FALSE,
+      'mailto' =>
+      [
+        'subject' => '[node:title]',
+        'body' => '[share:url]',
+      ],
+    ] + parent::defaults();
+  }
+
+  /**
```

Comment:
Rendering happens differently based on whether `mailto:` sharing is activated or not